app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/received/ReceivedScreen.kt

```diff
@@ -17,36 +17,84 @@
 import androidx.compose.material3.Text
 import androidx.compose.material3.adaptive.navigationsuite.NavigationSuiteType
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.infomaniak.swisstransfer.R
+import com.infomaniak.swisstransfer.ui.components.BrandTobAppBar
+import com.infomaniak.swisstransfer.ui.components.EmptyState
 import com.infomaniak.swisstransfer.ui.components.NewTransferFab
 import com.infomaniak.swisstransfer.ui.components.NewTransferFabType
+import com.infomaniak.swisstransfer.ui.images.AppImages.AppIllus
+import com.infomaniak.swisstransfer.ui.images.illus.ArrowRightCurved
+import com.infomaniak.swisstransfer.ui.images.illus.MascotSearching
 import com.infomaniak.swisstransfer.ui.screen.main.LocalNavType
+import com.infomaniak.swisstransfer.ui.theme.Margin
 import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
 import com.infomaniak.swisstransfer.ui.utils.PreviewMobile
 import com.infomaniak.swisstransfer.ui.utils.PreviewTablet
 
 @Composable
 fun ReceivedScreen(navigateToDetails: (transferId: Int) -> Unit) {
-    ReceivedScreen(navType = LocalNavType.current)
+    SwissTransferTheme {
+        ReceivedScreen(navType = LocalNavType.current)
+    }
 }
 
 @Composable
 private fun ReceivedScreen(navType: NavigationSuiteType) {
     Scaffold(
+        topBar = { BrandTobAppBar() },
         floatingActionButton = {
-            if (navType == NavigationSuiteType.NavigationBar) NewTransferFab(newTransferFabType = NewTransferFabType.BOTTOM_BAR)
+            ConstraintLayout {
+                val (text, icon, fab) = createRefs()
+
+                Text(
+                    text = LocalContext.current.getString(R.string.firstTransferDescription),
+                    style = SwissTransferTheme.typography.bodyRegular,
+                    color = SwissTransferTheme.colors.secondaryTextColor,
+                    modifier = Modifier
+                        .padding(PaddingValues(top = Margin.XLarge))
+                        .constrainAs(text) {
+                            bottom.linkTo(icon.top, Margin.Small)
+                            end.linkTo(icon.end, Margin.Small)
+                        },
+                )
+
+                Icon(
+                    modifier = Modifier
+                        .constrainAs(icon) {
+                            top.linkTo(fab.top)
+                            bottom.linkTo(fab.bottom, Margin.Small)
+                            end.linkTo(fab.start, Margin.Medium)
+                        },
+                    imageVector = AppIllus.ArrowRightCurved,
+                    contentDescription = null,
+                )
+
+                NewTransferFab(
+                    modifier = Modifier.constrainAs(fab) { },
+                    newTransferFabType = NewTransferFabType.BOTTOM_BAR,
+                )
+            }
```

Comment:
Extract to a new Component file, ex: `.ui.screen.main.received.ReceivedEmptyFab`